utils/releaser/src/ReleaseWorker/Release/MergeReleaseCandidateBranchReleaseWorker.php

```diff
@@ -33,9 +33,12 @@ public function getPriority(): int
      */
     public function work(Version $version): void
     {
-        $this->symfonyStyle->note('You need to create a merge commit, see https://docs.shopsys.com/en/latest/contributing/merging-on-github/ for detailed instructions.');
-        $this->symfonyStyle->warning(sprintf('If you are creating the merge commit locally, do not forget to push the "%s" branch!', $this->initialBranchName));
-        $this->confirm(sprintf('Confirm "%s" branch was merged to "%s"', $this->createBranchName($version), $this->initialBranchName));
+        $this->symfonyStyle->note('You need to create a merge commit locally, see https://docs.shopsys.com/en/latest/contributing/merging-on-github/ for detailed instructions.');
```

Comment:
If you need to create a merge commit locally, why do we provide link for docs about merging on github? :D

Reply:
removed